app/scripts/controllers/transactions/index.js

```diff
@@ -607,12 +584,13 @@ export default class TransactionController extends EventEmitter {
       // sign transaction
       const rawTx = await this.signTransaction(txId);
       await this.publishTransaction(txId, rawTx);
+      this._trackTxEvent(txMeta, 'Transaction Approved');
```

Comment:
Let's lift the strings for these events into constants at the top of this file as its unlikely they'll need to be used/referenced outside of this.

Reply:
so const TRANSACTION_APPROVED = 'Transaction Approved'

Reply:
add a structure the top containing string values 👍